browser/widevine/widevine_utils.cc

```diff
@@ -149,8 +182,18 @@ void MigrateWidevinePrefs(PrefService* prefs) {
   // they were explicitly set by primary prefs' value. After that, we don't
   // need to try migration again and prefs from profiles are already cleared.
   if (local_state->FindPreference(kWidevineEnabled)->IsDefaultValue()) {
-    local_state->SetBoolean(kWidevineEnabled,
-                            prefs->GetBoolean(kWidevineEnabled));
+    if (prefs->FindPreference(kWidevineEnabled)->IsDefaultValue()) {
+#if BUILDFLAG(IS_LINUX) && !defined(ARCH_CPU_X86_64)
+      // This is not actually a migration. But this point in the code is just
+      // too perfect to implement the following logic:
+      if (HasBundledWidevine()) {
+        local_state->SetDefaultPrefValue(kWidevineEnabled, base::Value(true));
+      }
```

Comment:
How about deleting MigrateWidevinePrefs() instead of adding more code to it because this migration code was added almost 3y ago. I think we could safely delete all methods that having Migrate or Obsolete in their name.
Maybe this code could be added in `RegisterWidevineLocalstatePrefs()`.